test/unit/specs/components/ActionModal/components/__snapshots__/TableInvoice.spec.js.snap

```diff
@@ -22,7 +25,7 @@ exports[`TableInvoice should show the transaction invoice summary 1`] = `
        
       <span>
         
-        0.030864
+        NaN
```

Comment:
this looks incorrect

Reply:
Thanks. The test wasn't updated to account for the new props.